pynest/examples/twoneurons.py

```diff
@@ -22,32 +22,61 @@
 """Two neuron example
 ----------------------------
 
+This script simulates two connected pre- and postsynaptic neurons.
+The presynaptic neuron receives a constant external current.
+And, the membrane potential of both neurons are recorded.
```

Comment:
Isn't starting a sentence with `And,` bad practice? I suggest joining it with the previous sentence.
```suggestion
This script simulates two connected pre- and postsynaptic neurons.
The presynaptic neuron receives a constant external current,
and the membrane potential of both neurons are recorded.
```